src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -297,6 +297,27 @@ void GeneralPage::initLayout(GeneralPageView &layout)
         false,
         "Specify how Chatterino will handle messages that exceed Twitch "
         "message limits");
+    layout.addDropdown<std::underlying_type<UsernameRightClickBehavior>::type>(
```

Comment:
The settings are confusing to me since they don't really say what right-click behaviour actually does (specifically the Reply with modifier & Mention with modifier). They only say what action is taken with the modifier.
It might be better to split this into two settings:
Username right-click behaviour: ... reply/mention
Username modifier right-click behaviour: ... reply/mention

Reply:
I've split this into two settings as you said. I also added an option to do nothing in both cases.